frontend/src/scenes/annotations/index.tsx

```diff
@@ -100,14 +100,20 @@ export function Annotations(): JSX.Element {
                 title="Annotations"
                 caption="Here you can add organization- and project-wide annotations. Dashboard-specific ones can be added directly in the dashboard."
                 buttons={
-                    <LemonButton type="primary" data-attr="create-annotation" onClick={(): void => setOpen(true)}>
+                    <LemonButton
+                        type="primary"
+                        data-attr="create-annotation"
+                        data-tooltip="annotations-new-button"
```

Comment:
Could we just reuse `data-attr` instead of adding `data-tooltip`? 🤔 Seems like they serve a very similar purpose